rpc/cloudagent/compute/availabilityzone/moc_cloudagent_availabilityzone.proto

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache v2.0 license.
+
+syntax = "proto3";
+option go_package = "github.com/microsoft/moc/rpc/cloudagent/compute";
+package moc.cloudagent.compute;
+
+import "google/protobuf/wrappers.proto";
+import "moc_common_common.proto";
+
+message AvailabilityZoneRequest {
+	repeated AvailabilityZone AvailabilityZones = 1;
+	Operation OperationType = 2;
+}
+
+message AvailabilityZoneResponse {
+	repeated AvailabilityZone AvailabilityZones = 1;
+	google.protobuf.BoolValue Result = 2;
+	string  Error = 3;
+}
+
+message VirtualMachineRef {
+	string groupName = 1;
+	string name = 2;
+	string realizedName = 3;
+}
+
+message AvailabilityZone {
+    string name = 1;
+    string id = 2;
+    string locationName = 3;
+	repeated string nodes = 4;
```

Comment:
fix indentation?

Reply:
image
This is showing fine in my local, not sure why it is showing differently here, tried multiple times fixing this but for some reason it is showing discrepancy between github and my local

Reply:
Probably spaces vs tab indentation. You can delete space before start of string and try tab indenting ?

Reply:
I tried the same with tab and spaces and it still shows the same indentation issue